diktat-cli/src/test/resources/test/smoke/src/main/kotlin/ClassKdocExpected.kt

```diff
@@ -0,0 +1,332 @@
+package com.saveourtool.diktat
+
+/**
+ * kdoc
+ * class
+ * comment
+ *
```

Comment:
Can you please check our codestyle document. Do we mention this newline?

Reply:
No, added note